content/sensu-go/6.1/operations/maintain-sensu/troubleshoot.md

```diff
@@ -379,6 +379,62 @@ Since the `lsb_release` package is not installed, the agent will not be able to
 
 To resolve this problem, install the [`lsb_release` package][8] for your Linux distribution.
 
+## Investigate etcd cluster status
+
+Some issues require investigating the state of the etcd cluster or data stored within etcd. In many such cases, this best accomplished by using the `etcdctl` tool to query and manage the etcd database.
+
+### Obtaining etcdctl
+
+Sensu official packages do not include the etcdctl executable. so you'll need to obtain it from a compatible etcd release.
+
+### Configure etcdctl environment variables
+
+```
+export ETCDCTL_API=3
+export ETCDCTL_CACERT=/etc/sensu/ca.pem
+export ETCDCTL_CERT=/rpool/data/subvol-112-disk-0/etc/sensu/etcd-client.crt
+export ETCDCTL_KEY=/rpool/data/subvol-112-disk-0/etc/sensu/etcd-client.key
+export ETCDCTL_ENDPOINTS="https://backend01:2379,https://backend02:2379,https://backend03:2379"
+```
+
+### See some stuff
+
+```
+# get status
+etcdctl --endpoints https://etcd.example.com:2379 endpoint status
+# list alarms
+etcdctl --endpoints https://etcd.example.com:2379 alarm list
+# clear alarms
+etcdctl --endpoints https://etcd.example.com:2379 alarm dearm
```

Comment:
GitHub won't let me add this as a suggestion (I think formatting is messing it up) but I suggest moving the `clear alarms` command to a separate section since it is doing something rather than just listing something:

```
### Clear etcd alarms

To clear etcd alarms:
```

Another possibility is to have a section `### View etcd cluster status` and then separate the alarms stuff `### List and clear etcd alarms`